src/seedu/addressbook/data/person/Email.java

```diff
@@ -58,4 +56,5 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    */
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.